### DISCUSSION TITLE
The shopify-partners has a cool component, but I find that the components in the polaris library just don't seem to do it

### DISCUSSION THREAD

**Discussion by transon-top** [Shopify/polaris-react#6655] [Q&A] [answered]
### offical rendering
![image](https://user-images.githubusercontent.com/68772719/179458654-54c964d7-217f-447e-9dac-8d428fe0fb83.png)
I would like to know which libraries implement this drag and drop feature. Because there are too many drag libraries. Thank you for your answer.  : )

**Comment by alex-page** [comment, marked as answer]
Hey @quanscheng this is built [dndkit](https://dndkit.com/). We aren't planning to add this functionality to Polaris. I do understand having docs in one place is very useful.
